src/helpers/paper-util.js

```diff
@@ -94,8 +94,18 @@ const PaperUtil = {
     const initialize = (): any => {
       paper.setup(canvas)
 
-      // Move View to be centered on 0,0
-      paper.view.transform(new paper.Matrix(1,0,0,-1,paper.view.center.x, paper.view.center.y))
+      // Move View to be centered on 0,0 of the Grid which is determined by
+      // the GraphSettings (not always the middle of the canvas)
+      // To do that we apply a translation to the paper view with the ratio
+      // of GridMinX * GridWidth and same for Y axis
```

Comment:
Shouldn't this be division, not multiplication? `GridMinX / GridWidth`, not `GridMinX * GridWidth`. That's what the code does.